draft-ietf-ppm-dap.md

```diff
@@ -337,99 +342,99 @@ The overall system architecture is shown in {{pa-topology}}.
                     |            |
                     +------------+
 ~~~~
-{: #pa-topology title="System Architecture"}
+{: #dap-topology title="System Architecture"}
 
 [[OPEN ISSUE: This shows two helpers, but the document only allows one for now.
 https://github.com/ietf-wg-ppm/draft-ietf-ppm-dap/issues/117]]
 
-
 The main participants in the protocol are as follows:
 
 Collector:
-: The entity which wants to take the measurement and ultimately receives the
-  results. Any given measurement will have a single collector.
+: The entity which wants to take aggregate the measurements generated by the
```

Comment:
```suggestion
: The entity which wants to obtain the aggregate of the measurements generated by the
```
It'd be awkward to suggest that the collector does the aggregating, because that's the aggregators' job.